tests/test_omnik.py

```diff
@@ -236,13 +239,12 @@ async def test_unexpected_response(aresponses: ResponsesMockServer) -> None:
         aresponses.Response(text="Give me energy!", status=200),
     )
 
-    async with aiohttp.ClientSession() as session:
+    async with ClientSession() as session:
         client = OmnikInverter(host="example.com", session=session)
         with pytest.raises(OmnikInverterError):
             assert await client.request("test")
 
 
-@pytest.mark.asyncio
 async def test_tcp_serial_number_unset() -> None:
```

Comment:
Should move this to the TCP test file?

Reply:
Yes, I think you only moved the `asynctest` cases initially :)